lib/compiler.js

```diff
@@ -28,10 +28,12 @@ function getDeps(requires) {
     globalNames: [],
     commonJS: {},
     commonJSNames: [],
-    ym: [],
-    ymVars: [],
-    ymVarsString: '',
+    ym: {},
+    ymNames: [],
     ymLibs: [],
+    ymVars: '',
```

Comment:
Why we need both `ymVars` and `ymNames`?
Previuosly, I used only `ymVars: []` and  `ymVars.join(', ')`

So many variables… I’m try to reduce their number.